### ISSUE TITLE
Component: Table

### ISSUE THREAD

**Issue by zozo8** [primefaces/primeng#12280]
### Describe the bug

When using p-table with stateStorage and resizableColumns the table is being shrinked into container size after refresh - in my p-card. This happens only after restoring state from local storage, so make sure you will resize any column then refresh the view.
I do believe it may be connected with adding style="width: XXXXpx;" property for first div under the p-table
Are there any known workarounds for that issue? Or maybe I've configured something wrongly.
Thank you in advance for helping me out.
Here is the screenshot of the issue:
Before reload:
![image](https://user-images.githubusercontent.com/50959052/203790452-9c5e7f52-23a1-4ec7-a809-a1368ba56f12.png)

After reload:
![image](https://user-images.githubusercontent.com/50959052/203790522-503bb8ab-f17d-4369-ac26-85cabbe163ae.png)

The expected behaviour is to make same column widths before and after reload.

### Environment

Windows 10 or 11

### Reproducer

https://stackblitz.com/edit/primeng-tablestate-demo-kjwedr?file=src%2Fapp%2Fapp.module.ts,src%2Fapp%2Fapp.component.html

### Angular version

14.2.9

### PrimeNG version

14.2.2

### Build / Runtime

Angular CLI App

### Language

TypeScript

### Node version (for AoT issues node --version)

0.0.0

### Browser(s)

Chrome 107

### Steps to reproduce the behavior

1. Expand any column(s) to make whole table wider than container and make horizontal scroll visible
2. Refresh view
3. Observe columns are smaller than before refresh

### Expected behavior

Columns should be the same widths before and after refresh

**Comment by mertsincan** [comment]
Hi,

So sorry for the delayed response! Improvements have been made to many components recently, both in terms of performance and enhancement. Therefore, this improvement may have been developed in another issue ticket without realizing it. You can check this in the documentation. If there is no improvement on this, can you reopen the issue so we can include it in our roadmap?
Please don't forget to add your feedback as a comment after reopening the issue. These will be taken into account by us and will contribute to the development of this feature. Thanks a lot for your understanding!

Best Regards,
